Update bounds for GHC 8.6

fosskers opened the pull request:
Here is a version of a `stack.yaml` which can be used to test this:
```yaml
# Let's try to keep resolver at the first day of the month
# resolver: nightly-2018-07-08
resolver: nightly-2018-10-01
packages:
- servant-client/
- servant-client-core/
- servant-docs/
- servant-foreign/
- servant-server/
- servant/

extra-deps:
  - control-monad-omega-0.3.1
  - cryptonite-0.25
  - http-client-0.5.13.1
  - http-media-0.7.1.3
  - memory-0.14.18
  - mime-types-0.1.0.8
  - wai-app-static-3.1.6.2
  - network-2.7.0.2@sha256:7cb8d7bc2080be946cf87890c807b12dae9c59314fa030b2687999100ce60512,2845

# allow-newer: true # ignores all bounds, that's a sledgehammer
# - doc/tutorial/
```

fosskers commented:
How is the Travis config regenerated? Fields for testing w/ 8.6 would have to be added too.

phadej commented:
- You need to disable `servant-auth` cookbook from CI, they won't build with GHC-8.6.
- note: `.travis.yml` is autogenerated, don't update it by hand. use the generator.

fosskers commented:
I read the README, re: regenerating. I'll add that soon.

fosskers commented:
I wish subsequent PR pushed would cancel previous CI builds.

fosskers commented:
Not really sure what 8.6 is failing there...
```
...
Finished in 0.3275 seconds
33 examples, 0 failures
Test suite spec: PASS
Test suite logged to:
/tmp/dist-test.Jo0m/dist-newstyle/build/x86_64-linux/ghc-8.6.1/servant-client-0.14/t/spec/test/servant-client-0.14-spec.log
1 of 1 test suites (1 of 1 test cases) passed.
Configuring test suite 'readme' for servant-client-0.14..
Preprocessing test suite 'readme' for servant-client-0.14..
Building test suite 'readme' for servant-client-0.14..
[1 of 1] Compiling Main             ( README.lhs, /tmp/dist-test.Jo0m/dist-newstyle/build/x86_64-linux/ghc-8.6.1/servant-client-0.14/t/readme/build/readme/readme-tmp/Main.o )
Linking /tmp/dist-test.Jo0m/dist-newstyle/build/x86_64-linux/ghc-8.6.1/servant-client-0.14/t/readme/build/readme/readme ...
Running 1 test suites...
Test suite readme: RUNNING...
Error: ConnectionError "HttpExceptionRequest Request {\n  host                 = \"localhost\"\n  port                 = 8081\n  secure               = False\n  requestHeaders       = [(\"Accept\",\"application/json;charset=utf-8,application/json\")]\n  path                 = \"/books\"\n  queryString          = \"\"\n  method               = \"GET\"\n  proxy                = Nothing\n  rawBody              = False\n  redirectCount        = 10\n  responseTimeout      = ResponseTimeoutDefault\n  requestVersion       = HTTP/1.1\n}\n (ConnectionFailure Network.Socket.connect: <socket: 3>: does not exist (Connection refused))"
Test suite readme: PASS
Test suite logged to:
/tmp/dist-test.Jo0m/dist-newstyle/build/x86_64-linux/ghc-8.6.1/servant-client-0.14/t/readme/test/servant-client-0.14-readme.log
1 of 1 test suites (1 of 1 test cases) passed.
The command "if [ "x$TEST" = "x--enable-tests" ]; then cabal new-test -w ${HC} ${TEST} ${BENCH} all; fi" exited with 0.
0.00s
haddock
Haddock...
22.21s$ if $HADDOCK; then cabal new-haddock -w ${HC} ${TEST} ${BENCH} all; else echo "Skipping haddock generation";fi
Build profile: -w ghc-8.6.1 -O1
In order, the following will be built (use -v for more details):
 - servant-0.14.1 (lib:servant) (ephemeral targets)
 - servant-server-0.14.1 (dependency rebuilt)
 - servant-foreign-0.11.1 (lib) (dependency rebuilt)
 - servant-docs-0.11.2 (lib) (dependency rebuilt)
 - servant-client-core-0.14.1 (lib) (dependency rebuilt)
 - servant-client-0.14 (lib) (dependency rebuilt)
Preprocessing library for servant-0.14.1..
Building library for servant-0.14.1..
Preprocessing library for servant-0.14.1..
Running Haddock on library for servant-0.14.1..
Warning: The documentation for the following packages are not installed. No
links will be generated to these packages: StateVar-1.1.1.1, aeson-1.4.1.0,
attoparsec-0.13.2.2, attoparsec-iso8601-1.0.1.0, base-compat-0.10.4,
base-orphans-0.8, bifunctors-5.5.3, case-insensitive-1.2.0.11, comonad-5.0.4,
contravariant-1.5, cookie-0.4.4, data-default-class-0.1.2.0, distributive-0.6,
dlist-0.8.0.5, hashable-1.2.7.0, http-api-data-0.3.10, http-media-0.7.1.3,
http-types-0.12.2, integer-logarithms-1.0.2.2, mmorph-1.1.2,
network-uri-2.6.1.0, primitive-0.6.4.0, random-1.1, scientific-0.3.6.2,
semigroups-0.18.5, singleton-bool-0.1.4, string-conversions-0.4.0.1,
tagged-0.8.6, th-abstraction-0.2.8.0, time-locale-compat-0.1.1.5,
transformers-compat-0.6.2, unordered-containers-0.2.9.0, utf8-string-1.0.1.1,
uuid-types-1.0.3, vault-0.3.1.2, vector-0.12.0.1
Haddock coverage:
  50% (  1 /  2) in 'Servant.API.Alternative'
  Missing documentation for:
    Module header
  67% (  2 /  3) in 'Servant.API.BasicAuth'
  Missing documentation for:
    Module header
Warning: 'Description' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
  75% (  3 /  4) in 'Servant.API.Capture'
  Missing documentation for:
    Module header
Warning: 'ReqBody' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'eitherDecode' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
  48% ( 10 / 21) in 'Servant.API.ContentTypes'
  Missing documentation for:
    JSON (src/Servant/API/ContentTypes.hs:106)
    PlainText (src/Servant/API/ContentTypes.hs:107)
    FormUrlEncoded (src/Servant/API/ContentTypes.hs:108)
    OctetStream (src/Servant/API/ContentTypes.hs:109)
    AcceptHeader (src/Servant/API/ContentTypes.hs:153)
    AllCTRender (src/Servant/API/ContentTypes.hs:176)
    AllCTUnrender (src/Servant/API/ContentTypes.hs:230)
    AllMime (src/Servant/API/ContentTypes.hs:249)
    AllMimeRender (src/Servant/API/ContentTypes.hs:267)
    AllMimeUnrender (src/Servant/API/ContentTypes.hs:307)
    canHandleAcceptH (src/Servant/API/ContentTypes.hs:261)
  88% (  7 /  8) in 'Servant.API.Description'
  Missing documentation for:
    Module header
Warning: ':<|>' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
  50% (  1 /  2) in 'Servant.API.Empty'
  Missing documentation for:
    Module header
  50% (  1 /  2) in 'Servant.API.Experimental.Auth'
  Missing documentation for:
    Module header
Warning: 'route' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'mode' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'routes' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'AsServerT' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'ServerT' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'generated' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
  86% ( 12 / 14) in 'Servant.API.Generic'
  Missing documentation for:
    ToServantApi (src/Servant/API/Generic.hs:93)
    GServantProduct (src/Servant/API/Generic.hs:128)
Warning: Couldn't find .haddock for export HttpVersion
  33% (  1 /  3) in 'Servant.API.HttpVersion'
  Missing documentation for:
    Module header
    HttpVersion
  67% (  2 /  3) in 'Servant.API.IsSecure'
  Missing documentation for:
    Module header
Warning: 'RequiredAgument' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
  94% ( 16 / 17) in 'Servant.API.Modifiers'
  Missing documentation for:
    Module header
  33% (  1 /  3) in 'Servant.API.Header'
  Missing documentation for:
    Module header
    Header' (src/Servant/API/Header.hs:26)
  80% (  4 /  5) in 'Servant.API.QueryParam'
  Missing documentation for:
    Module header
Warning: 'Application' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'pathInfo' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
  50% (  1 /  2) in 'Servant.API.Raw'
  Missing documentation for:
    Module header
  67% (  2 /  3) in 'Servant.API.RemoteHost'
    Module header
  67% (  2 /  3) in 'Servant.API.ReqBody'
  Missing documentation for:
    Module header
  40% (  4 / 10) in 'Servant.API.ResponseHeaders'
  Missing documentation for:
    ResponseHeader (src/Servant/API/ResponseHeaders.hs:60)
    AddHeader (src/Servant/API/ResponseHeaders.hs:140)
    BuildHeadersTo (src/Servant/API/ResponseHeaders.hs:75)
    GetHeaders (src/Servant/API/ResponseHeaders.hs:98)
    HeaderValMap (src/Servant/API/ResponseHeaders.hs:70)
    HList (src/Servant/API/ResponseHeaders.hs:66)
  84% ( 16 / 19) in 'Servant.API.Stream'
  Missing documentation for:
    Module header
    StreamGet (src/Servant/API/Stream.hs:70)
    StreamPost (src/Servant/API/Stream.hs:71)
  50% (  1 /  2) in 'Servant.API.Sub'
  Missing documentation for:
    Module header
Warning: Couldn't find .haddock for export Vault
  33% (  1 /  3) in 'Servant.API.Vault'
  Missing documentation for:
    Module header
    Vault
Warning: Couldn't find .haddock for export StdMethod
  91% ( 29 / 32) in 'Servant.API.Verbs'
  Missing documentation for:
    Module header
    ReflectMethod (src/Servant/API/Verbs.hs:158)
    StdMethod
  91% ( 20 / 22) in 'Servant.API.TypeLevel'
  Missing documentation for:
    IsSubList (src/Servant/API/TypeLevel.hs:201)
    ElemGo (src/Servant/API/TypeLevel.hs:217)
Warning: 'ReqBody' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'Get' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'Context' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
  50% (  1 /  2) in 'Servant.API.WithNamedContext'
  Missing documentation for:
    Module header
Warning: 'API' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'QueryParam' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'Capture' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'endpoint' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'api' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: Couldn't find .haddock for export URI
  88% ( 22 / 25) in 'Servant.Links'
  Missing documentation for:
    URI
    linkSegments (src/Servant/Links.hs:205)
    linkQueryParams (src/Servant/Links.hs:208)
Warning: 'Header' is ambiguous. It is defined
    * at src/Servant/API/ResponseHeaders.hs:61:7
    * at src/Servant/API/Header.hs:24:1
    You may be able to disambiguate the identifier by qualifying it or
    by hiding some imports.
    Defaulting to 'Header' defined at src/Servant/API/ResponseHeaders.hs:60:1
Warning: Couldn't find .haddock for export HttpVersion
Warning: Couldn't find .haddock for export Vault
Warning: Couldn't find .haddock for export StdMethod
Warning: Couldn't find .haddock for export FromHttpApiData
Warning: Couldn't find .haddock for export ToHttpApiData
Warning: Couldn't find .haddock for export URI
Warning: Couldn't find .haddock for export SBool
Warning: Couldn't find .haddock for export SBoolI
  83% (108 /130) in 'Servant.API'
  Missing documentation for:
    Module header
    Header' (src/Servant/API/Header.hs:26)
    HttpVersion
    Vault
    StdMethod
    ReflectMethod (src/Servant/API/Verbs.hs:158)
    StreamPost (src/Servant/API/Stream.hs:71)
    StreamGet (src/Servant/API/Stream.hs:70)
    OctetStream (src/Servant/API/ContentTypes.hs:109)
    FormUrlEncoded (src/Servant/API/ContentTypes.hs:108)
    PlainText (src/Servant/API/ContentTypes.hs:107)
    JSON (src/Servant/API/ContentTypes.hs:106)
    AddHeader (src/Servant/API/ResponseHeaders.hs:140)
    GetHeaders (src/Servant/API/ResponseHeaders.hs:98)
    BuildHeadersTo (src/Servant/API/ResponseHeaders.hs:75)
    HList (src/Servant/API/ResponseHeaders.hs:66)
    ResponseHeader (src/Servant/API/ResponseHeaders.hs:60)
    FromHttpApiData
    ToHttpApiData
    URI
    SBool
    SBoolI
  17% (  1 /  6) in 'Servant.API.Internal.Test.ComprehensiveAPI'
  Missing documentation for:
    GET (src/Servant/API/Internal/Test/ComprehensiveAPI.hs:13)
    ComprehensiveAPI (src/Servant/API/Internal/Test/ComprehensiveAPI.hs:15)
    comprehensiveAPI (src/Servant/API/Internal/Test/ComprehensiveAPI.hs:19)
    ComprehensiveAPIWithoutRaw (src/Servant/API/Internal/Test/ComprehensiveAPI.hs:22)
    comprehensiveAPIWithoutRaw (src/Servant/API/Internal/Test/ComprehensiveAPI.hs:49)
  50% (  1 /  2) in 'Servant.Utils.Links'
  Missing documentation for:
    Module header
Warning: Servant.API.Alternative: could not find link destinations for:
    Biapplicative bipure <<*>> biliftA2 *>> <<*
Warning: Servant.API.ContentTypes: could not find link destinations for:
    MediaType FromJSON ToJSON encode FromForm ToForm
Warning: Servant.API.Generic: could not find link destinations for:
    GToServant gtoServant gfromServant Value Form Tagged
Warning: Servant.API.Modifiers: could not find link destinations for:
    addOptionalHeader
Warning: Servant.API.Header: could not find link destinations for:
    addOptionalHeader
Warning: Servant.API.ResponseHeaders: could not find link destinations for:
    addOptionalHeader GetHeaders' Header GetHeadersFromHList
Warning: Servant.API.Vault: could not find link destinations for:
    Vault
Warning: Servant.API.Verbs: could not find link destinations for:
    Method unsafeIndex unsafeRangeSize
Warning: Servant.Links: could not find link destinations for:
    URIAuth
Warning: Servant.API: could not find link destinations for:
    Biapplicative bipure <<*>> biliftA2 *>> <<* addOptionalHeader Vault unsafeIndex unsafeRangeSize Method MediaType FromForm FromJSON ToForm ToJSON encode Header GetHeadersFromHList GetHeaders' UUID SetCookie LenientData URIAuth
Documentation created:
/tmp/dist-test.Jo0m/dist-newstyle/build/x86_64-linux/ghc-8.6.1/servant-0.14.1/doc/html/servant/index.html
Preprocessing library for servant-server-0.14.1..
Running Haddock on library for servant-server-0.14.1..
Warning: The documentation for the following packages are not installed. No
links will be generated to these packages: StateVar-1.1.1.1, aeson-1.4.1.0,
ansi-terminal-0.8.1, ansi-wl-pprint-0.6.8.2, appar-0.1.4, async-2.2.1,
attoparsec-0.13.2.2, attoparsec-iso8601-1.0.1.0, auto-update-0.1.4,
base-compat-0.10.4, base-orphans-0.8, base64-bytestring-1.0.0.1,
basement-0.0.8, bifunctors-5.5.3, blaze-builder-0.4.1.0, blaze-html-0.9.1.1,
blaze-markup-0.8.2.2, bsb-http-chunked-0.0.0.4, byteorder-1.0.4,
bytestring-builder-0.10.8.2.0, case-insensitive-1.2.0.11, colour-2.3.4,
comonad-5.0.4, contravariant-1.5, cookie-0.4.4, cryptonite-0.25,
data-default-class-0.1.2.0, distributive-0.6, dlist-0.8.0.5, easy-file-0.2.2,
exceptions-0.10.0, fast-logger-2.4.11, file-embed-0.0.11, hashable-1.2.7.0,
http-api-data-0.3.10, http-date-0.0.8, http-media-0.7.1.3, http-types-0.12.2,
http2-1.6.3, integer-logarithms-1.0.2.2, iproute-1.7.5, memory-0.14.18,
mime-types-0.1.0.8, mmorph-1.1.2, monad-control-1.0.2.3, network-2.7.0.2,
network-uri-2.6.1.0, old-locale-1.0.0.7, old-time-1.1.0.3,
optparse-applicative-0.14.3.0, primitive-0.6.4.0, psqueues-0.2.7.0,
random-1.1, resourcet-1.2.1, scientific-0.3.6.2, semigroups-0.18.5,
simple-sendfile-0.2.27, singleton-bool-0.1.4, streaming-commons-0.2.1.0,
string-conversions-0.4.0.1, tagged-0.8.6, th-abstraction-0.2.8.0,
time-locale-compat-0.1.1.5, transformers-base-0.4.5.2,
transformers-compat-0.6.2, unix-compat-0.5.1, unix-time-0.3.8,
unliftio-core-0.1.2.0, unordered-containers-0.2.9.0, utf8-string-1.0.1.1,
uuid-types-1.0.3, vault-0.3.1.2, vector-0.12.0.1, void-0.7.2, wai-3.2.1.2,
wai-app-static-3.1.6.2, wai-extra-3.0.24.3, wai-logger-2.3.2, warp-3.2.25,
word8-0.1.3, zlib-0.6.2
Haddock coverage:
Warning: 'hoistServer' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
  83% (  5 /  6) in 'Servant.Server.Internal.Context'
  Missing documentation for:
    Module header
  91% ( 32 / 35) in 'Servant.Server.Internal.ServantErr'
  Missing documentation for:
    Module header
    ServantErr (src/Servant/Server/Internal/ServantErr.hs:16)
    responseServantErr (src/Servant/Server/Internal/ServantErr.hs:24)
   0% (  0 /  3) in 'Servant.Server.Internal.Handler'
  Missing documentation for:
    Module header
    Handler (src/Servant/Server/Internal/Handler.hs:28)
    runHandler (src/Servant/Server/Internal/Handler.hs:47)
Warning: 'pathInfo' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
  74% ( 17 / 23) in 'Servant.Server.Internal.RoutingApplication'
  Missing documentation for:
    Module header
    RoutingApplication (src/Servant/Server/Internal/RoutingApplication.hs:36)
    RouteResultT (src/Servant/Server/Internal/RoutingApplication.hs:58)
    toApplication (src/Servant/Server/Internal/RoutingApplication.hs:96)
    liftRouteResult (src/Servant/Server/Internal/RoutingApplication.hs:212)
    runDelayedIO (src/Servant/Server/Internal/RoutingApplication.hs:227)
src/Servant/Server/Internal/Router.hs:10:1: warning: [-Wunused-imports]
    The import of ‘Data.Monoid’ is redundant
      except perhaps to import instances from ‘Data.Monoid’
    To import instances alone, use: import Data.Monoid()
   |
10 | import           Data.Monoid
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Warning: 'WithRequest' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
  73% ( 11 / 15) in 'Servant.Server.Internal.Router'
  Missing documentation for:
    Module header
    Router (src/Servant/Server/Internal/Router.hs:19)
    runRouterEnv (src/Servant/Server/Internal/Router.hs:154)
    worseHTTPCode (src/Servant/Server/Internal/Router.hs:211)
Warning: 'Authorization' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
  86% (  6 /  7) in 'Servant.Server.Internal.BasicAuth'
  Missing documentation for:
    Module header
Warning: 'Capture' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'Header' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'QueryParam' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'ReqBody' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'FromJSON' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'emptyAPIServer' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
  40% (  8 / 20) in 'Servant.Server.Internal'
  Missing documentation for:
    Module header
    HasServerArrowTypeError (src/Servant/Server/Internal.hs:801)
    HasServerArrowKindError (src/Servant/Server/Internal.hs:770)
    Server (src/Servant/Server/Internal.hs:117)
    HasServer (src/Servant/Server/Internal.hs:101)
    allowedMethodHead (src/Servant/Server/Internal.hs:215)
    allowedMethod (src/Servant/Server/Internal.hs:218)
    methodCheck (src/Servant/Server/Internal.hs:221)
    acceptCheck (src/Servant/Server/Internal.hs:233)
    methodRouter (src/Servant/Server/Internal.hs:238)
    streamRouter (src/Servant/Server/Internal.hs:307)
    ct_wildcard (src/Servant/Server/Internal.hs:715)
Warning: 'Raw' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: Couldn't find .haddock for export Application
Warning: Couldn't find .haddock for export Tagged
  87% ( 61 / 70) in 'Servant.Server'
  Missing documentation for:
    serveWithContext (src/Servant/Server.hs:132)
    toApplication (src/Servant/Server/Internal/RoutingApplication.hs:96)
    HasServer (src/Servant/Server/Internal.hs:101)
    Server (src/Servant/Server/Internal.hs:117)
    Handler (src/Servant/Server/Internal/Handler.hs:28)
    runHandler (src/Servant/Server/Internal/Handler.hs:47)
    ServantErr (src/Servant/Server/Internal/ServantErr.hs:16)
    Application
    Tagged
  67% (  4 /  6) in 'Servant.Server.Generic'
  Missing documentation for:
    AsServer (src/Servant/Server/Generic.hs:29)
    genericServerT (src/Servant/Server/Generic.hs:48)
 100% (  8 /  8) in 'Servant.Server.StaticFiles'
  91% ( 10 / 11) in 'Servant'
  Missing documentation for:
    Module header
Warning: 'User' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
  80% (  4 /  5) in 'Servant.Server.Experimental.Auth'
  Missing documentation for:
    Module header
  50% (  1 /  2) in 'Servant.Utils.StaticFiles'
  Missing documentation for:
    Module header
Warning: Servant.Server.Internal.ServantErr: could not find link destinations for:
    Header Response
Warning: Servant.Server.Internal.Handler: could not find link destinations for:
    MonadCatch catch MonadThrow throwM MonadBase liftBase MonadBaseControl liftBaseWith RunInBase restoreM StM
Warning: Servant.Server.Internal.RoutingApplication: could not find link destinations for:
    Request Response ResponseReceived MonadTransControl liftWith Run restoreT StT MonadBase liftBase MonadBaseControl liftBaseWith RunInBase restoreM StM MonadThrow throwM ComposeSt ResourceT MonadResource liftResourceT
Warning: Servant.Server.Internal.Router: could not find link destinations for:
    Response
Warning: Servant.Server.Internal.BasicAuth: could not find link destinations for:
    Header Request
Warning: Servant.Server.Internal: could not find link destinations for:
    Request Method HeaderName Status
Warning: Servant.Server: could not find link destinations for:
    Request MonadCatch catch MonadThrow throwM MonadBase liftBase MonadBaseControl liftBaseWith RunInBase restoreM StM Response Header ResponseReceived unsafeIndex unsafeRangeSize
Warning: Servant.Server.StaticFiles: could not find link destinations for:
    defaultWebAppSettings StaticSettings ETagLookup webAppSettingsWithLookup defaultFileServerSettings embeddedSettings staticApp
Warning: Servant: could not find link destinations for:
    Hashable1 liftHashWithSalt unsafeIndex unsafeRangeSize Hashable hashWithSalt hash
Warning: Servant.Server.Experimental.Auth: could not find link destinations for:
    Request
Documentation created:
/tmp/dist-test.Jo0m/dist-newstyle/build/x86_64-linux/ghc-8.6.1/servant-server-0.14.1/doc/html/servant-server/index.html
Preprocessing library for servant-foreign-0.11.1..
Running Haddock on library for servant-foreign-0.11.1..
Warning: The documentation for the following packages are not installed. No
links will be generated to these packages: StateVar-1.1.1.1, adjunctions-4.4,
aeson-1.4.1.0, attoparsec-0.13.2.2, attoparsec-iso8601-1.0.1.0,
base-compat-0.10.4, base-orphans-0.8, bifunctors-5.5.3, call-stack-0.1.0,
case-insensitive-1.2.0.11, comonad-5.0.4, contravariant-1.5, cookie-0.4.4,
data-default-class-0.1.2.0, distributive-0.6, dlist-0.8.0.5,
exceptions-0.10.0, free-5.1, hashable-1.2.7.0, http-api-data-0.3.10,
http-media-0.7.1.3, http-types-0.12.2, integer-logarithms-1.0.2.2,
invariant-0.5.1, kan-extensions-5.2, lens-4.17, mmorph-1.1.2,
network-uri-2.6.1.0, parallel-3.2.2.0, primitive-0.6.4.0, profunctors-5.3,
random-1.1, reflection-2.1.4, scientific-0.3.6.2, semigroupoids-5.3.1,
semigroups-0.18.5, singleton-bool-0.1.4, string-conversions-0.4.0.1,
tagged-0.8.6, th-abstraction-0.2.8.0, time-locale-compat-0.1.1.5,
transformers-base-0.4.5.2, transformers-compat-0.6.2,
unordered-containers-0.2.9.0, utf8-string-1.0.1.1, uuid-types-1.0.3,
vault-0.3.1.2, vector-0.12.0.1, void-0.7.2
Haddock coverage:
   9% (  4 / 47) in 'Servant.Foreign.Internal'
  Missing documentation for:
    FunctionName (src/Servant/Foreign/Internal.hs:43)
    _FunctionName (src/Servant/Foreign/Internal.hs:46)
    PathSegment (src/Servant/Foreign/Internal.hs:48)
    _PathSegment (src/Servant/Foreign/Internal.hs:51)
    Arg (src/Servant/Foreign/Internal.hs:53)
    argType (src/Servant/Foreign/Internal.hs:58)
    argName (src/Servant/Foreign/Internal.hs:58)
    argPath (src/Servant/Foreign/Internal.hs:60)
    SegmentType (src/Servant/Foreign/Internal.hs:63)
    _Cap (src/Servant/Foreign/Internal.hs:70)
    _Static (src/Servant/Foreign/Internal.hs:70)
    Segment (src/Servant/Foreign/Internal.hs:72)
    _Segment (src/Servant/Foreign/Internal.hs:75)
    isCapture (src/Servant/Foreign/Internal.hs:77)
    captureArg (src/Servant/Foreign/Internal.hs:81)
    Path (src/Servant/Foreign/Internal.hs:85)
    ArgType (src/Servant/Foreign/Internal.hs:87)
    _List (src/Servant/Foreign/Internal.hs:93)
    _Flag (src/Servant/Foreign/Internal.hs:93)
    _Normal (src/Servant/Foreign/Internal.hs:93)
    QueryArg (src/Servant/Foreign/Internal.hs:95)
    queryArgType (src/Servant/Foreign/Internal.hs:101)
    queryArgName (src/Servant/Foreign/Internal.hs:101)
    HeaderArg (src/Servant/Foreign/Internal.hs:103)
    headerPattern (src/Servant/Foreign/Internal.hs:111)
    headerArg (src/Servant/Foreign/Internal.hs:111)
    _ReplaceHeaderArg (src/Servant/Foreign/Internal.hs:113)
    _HeaderArg (src/Servant/Foreign/Internal.hs:113)
    Url (src/Servant/Foreign/Internal.hs:115)
    defUrl (src/Servant/Foreign/Internal.hs:121)
    queryStr (src/Servant/Foreign/Internal.hs:124)
    path (src/Servant/Foreign/Internal.hs:124)
    Req (src/Servant/Foreign/Internal.hs:126)
    reqUrl (src/Servant/Foreign/Internal.hs:136)
    reqReturnType (src/Servant/Foreign/Internal.hs:136)
    reqMethod (src/Servant/Foreign/Internal.hs:136)
    reqHeaders (src/Servant/Foreign/Internal.hs:136)
    reqFuncName (src/Servant/Foreign/Internal.hs:136)
    reqBody (src/Servant/Foreign/Internal.hs:136)
    defReq (src/Servant/Foreign/Internal.hs:138)
    NoTypes (src/Servant/Foreign/Internal.hs:176)
    HasForeign (src/Servant/Foreign/Internal.hs:181)
    EmptyForeignAPI (src/Servant/Foreign/Internal.hs:193)
  43% (  3 /  7) in 'Servant.Foreign.Inflections'
  Missing documentation for:
    Module header
    concatCaseL (src/Servant/Foreign/Inflections.hs:23)
    snakeCaseL (src/Servant/Foreign/Inflections.hs:30)
    camelCaseL (src/Servant/Foreign/Inflections.hs:38)
  14% (  6 / 44) in 'Servant.Foreign'
  Missing documentation for:
    ArgType (src/Servant/Foreign/Internal.hs:87)
    HeaderArg (src/Servant/Foreign/Internal.hs:103)
    QueryArg (src/Servant/Foreign/Internal.hs:95)
    Req (src/Servant/Foreign/Internal.hs:126)
    Segment (src/Servant/Foreign/Internal.hs:72)
    SegmentType (src/Servant/Foreign/Internal.hs:63)
    Url (src/Servant/Foreign/Internal.hs:115)
    Path (src/Servant/Foreign/Internal.hs:85)
    Arg (src/Servant/Foreign/Internal.hs:53)
    FunctionName (src/Servant/Foreign/Internal.hs:43)
    PathSegment (src/Servant/Foreign/Internal.hs:48)
    argName (src/Servant/Foreign/Internal.hs:58)
    argType (src/Servant/Foreign/Internal.hs:58)
    argPath (src/Servant/Foreign/Internal.hs:60)
    reqUrl (src/Servant/Foreign/Internal.hs:136)
    reqMethod (src/Servant/Foreign/Internal.hs:136)
    reqHeaders (src/Servant/Foreign/Internal.hs:136)
    reqBody (src/Servant/Foreign/Internal.hs:136)
    reqReturnType (src/Servant/Foreign/Internal.hs:136)
    reqFuncName (src/Servant/Foreign/Internal.hs:136)
    path (src/Servant/Foreign/Internal.hs:124)
    queryStr (src/Servant/Foreign/Internal.hs:124)
    queryArgName (src/Servant/Foreign/Internal.hs:101)
    queryArgType (src/Servant/Foreign/Internal.hs:101)
    headerArg (src/Servant/Foreign/Internal.hs:111)
    _PathSegment (src/Servant/Foreign/Internal.hs:51)
    _HeaderArg (src/Servant/Foreign/Internal.hs:113)
    _ReplaceHeaderArg (src/Servant/Foreign/Internal.hs:113)
    _Static (src/Servant/Foreign/Internal.hs:70)
    _Cap (src/Servant/Foreign/Internal.hs:70)
    _Normal (src/Servant/Foreign/Internal.hs:93)
    _Flag (src/Servant/Foreign/Internal.hs:93)
    _List (src/Servant/Foreign/Internal.hs:93)
    HasForeign (src/Servant/Foreign/Internal.hs:181)
    NoTypes (src/Servant/Foreign/Internal.hs:176)
    captureArg (src/Servant/Foreign/Internal.hs:81)
    isCapture (src/Servant/Foreign/Internal.hs:77)
    defReq (src/Servant/Foreign/Internal.hs:138)
Warning: Servant.Foreign.Internal: could not find link destinations for:
    Iso' Lens Lens' Getter Prism Prism' Iso Traversal' Method
Warning: Servant.Foreign.Inflections: could not find link destinations for:
    Getter
Warning: Servant.Foreign: could not find link destinations for:
    Method Lens' Lens Getter Iso' Prism' Prism
Documentation created:
/tmp/dist-test.Jo0m/dist-newstyle/build/x86_64-linux/ghc-8.6.1/servant-foreign-0.11.1/doc/html/servant-foreign/index.html
Preprocessing library for servant-docs-0.11.2..
Running Haddock on library for servant-docs-0.11.2..
Warning: The documentation for the following packages are not installed. No
links will be generated to these packages: StateVar-1.1.1.1, adjunctions-4.4,
aeson-1.4.1.0, aeson-pretty-0.8.7, attoparsec-0.13.2.2,
attoparsec-iso8601-1.0.1.0, base-compat-0.10.4, base-orphans-0.8,
bifunctors-5.5.3, call-stack-0.1.0, case-insensitive-1.2.0.11, comonad-5.0.4,
contravariant-1.5, control-monad-omega-0.3.1, cookie-0.4.4,
data-default-class-0.1.2.0, distributive-0.6, dlist-0.8.0.5,
exceptions-0.10.0, free-5.1, hashable-1.2.7.0, http-api-data-0.3.10,
http-media-0.7.1.3, http-types-0.12.2, integer-logarithms-1.0.2.2,
invariant-0.5.1, kan-extensions-5.2, lens-4.17, mmorph-1.1.2,
network-uri-2.6.1.0, parallel-3.2.2.0, primitive-0.6.4.0, profunctors-5.3,
random-1.1, reflection-2.1.4, scientific-0.3.6.2, semigroupoids-5.3.1,
semigroups-0.18.5, singleton-bool-0.1.4, string-conversions-0.4.0.1,
tagged-0.8.6, th-abstraction-0.2.8.0, time-locale-compat-0.1.1.5,
transformers-base-0.4.5.2, transformers-compat-0.6.2,
unordered-containers-0.2.9.0, utf8-string-1.0.1.1, uuid-types-1.0.3,
vault-0.3.1.2, vector-0.12.0.1, void-0.7.2
Haddock coverage:
src/Servant/Docs/Internal.hs:869:11: warning: [-Wunused-local-binds]
    Defined but not used: ‘p’
    |
869 |           p = Proxy :: Proxy a
    |           ^
Warning: 'MediaType' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'DocInfo' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'Text' is ambiguous. It is defined
    * in ‘GHC.TypeLits’
    * in ‘Data.Text.Internal’
    You may be able to disambiguate the identifier by qualifying it or
    by hiding some imports.
    Defaulting to 'Text' defined in ‘Data.Text.Internal’
Warning: 'Omega' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
  56% ( 44 / 79) in 'Servant.Docs.Internal'
  Missing documentation for:
    Module header
    authIntro (src/Servant/Docs/Internal.hs:345)
    authDataRequired (src/Servant/Docs/Internal.hs:345)
    maxSamples (src/Servant/Docs/Internal.hs:346)
    apiIntros (src/Servant/Docs/Internal.hs:347)
    apiEndpoints (src/Servant/Docs/Internal.hs:347)
    path (src/Servant/Docs/Internal.hs:348)
    method (src/Servant/Docs/Internal.hs:348)
    capSymbol (src/Servant/Docs/Internal.hs:349)
    capDesc (src/Servant/Docs/Internal.hs:349)
    paramValues (src/Servant/Docs/Internal.hs:350)
    paramName (src/Servant/Docs/Internal.hs:350)
    paramKind (src/Servant/Docs/Internal.hs:350)
    paramDesc (src/Servant/Docs/Internal.hs:350)
    introTitle (src/Servant/Docs/Internal.hs:351)
    introBody (src/Servant/Docs/Internal.hs:351)
    noteTitle (src/Servant/Docs/Internal.hs:352)
    noteBody (src/Servant/Docs/Internal.hs:352)
    respTypes (src/Servant/Docs/Internal.hs:353)
    respStatus (src/Servant/Docs/Internal.hs:353)
    respHeaders (src/Servant/Docs/Internal.hs:353)
    respBody (src/Servant/Docs/Internal.hs:353)
    rqtypes (src/Servant/Docs/Internal.hs:354)
    rqbody (src/Servant/Docs/Internal.hs:354)
    response (src/Servant/Docs/Internal.hs:354)
    params (src/Servant/Docs/Internal.hs:354)
    notes (src/Servant/Docs/Internal.hs:354)
    mxParams (src/Servant/Docs/Internal.hs:354)
    headers (src/Servant/Docs/Internal.hs:354)
    captures (src/Servant/Docs/Internal.hs:354)
    authInfo (src/Servant/Docs/Internal.hs:354)
    responseExamples (src/Servant/Docs/Internal.hs:355)
    requestExamples (src/Servant/Docs/Internal.hs:355)
    notesHeading (src/Servant/Docs/Internal.hs:355)
    AllHeaderSamples (src/Servant/Docs/Internal.hs:507)
Warning: 'docs' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
  75% (  3 /  4) in 'Servant.Docs.Internal.Pretty'
  Missing documentation for:
    Module header
Warning: 'Proxy' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
  59% ( 45 / 76) in 'Servant.Docs'
  Missing documentation for:
    requestExamples (src/Servant/Docs/Internal.hs:355)
    responseExamples (src/Servant/Docs/Internal.hs:355)
    notesHeading (src/Servant/Docs/Internal.hs:355)
    maxSamples (src/Servant/Docs/Internal.hs:346)
    path (src/Servant/Docs/Internal.hs:348)
    method (src/Servant/Docs/Internal.hs:348)
    apiIntros (src/Servant/Docs/Internal.hs:347)
    apiEndpoints (src/Servant/Docs/Internal.hs:347)
    authIntro (src/Servant/Docs/Internal.hs:345)
    authDataRequired (src/Servant/Docs/Internal.hs:345)
    capSymbol (src/Servant/Docs/Internal.hs:349)
    capDesc (src/Servant/Docs/Internal.hs:349)
    paramName (src/Servant/Docs/Internal.hs:350)
    paramValues (src/Servant/Docs/Internal.hs:350)
    paramDesc (src/Servant/Docs/Internal.hs:350)
    paramKind (src/Servant/Docs/Internal.hs:350)
    noteTitle (src/Servant/Docs/Internal.hs:352)
    noteBody (src/Servant/Docs/Internal.hs:352)
    introTitle (src/Servant/Docs/Internal.hs:351)
    introBody (src/Servant/Docs/Internal.hs:351)
    respStatus (src/Servant/Docs/Internal.hs:353)
    respTypes (src/Servant/Docs/Internal.hs:353)
    respBody (src/Servant/Docs/Internal.hs:353)
    authInfo (src/Servant/Docs/Internal.hs:354)
    captures (src/Servant/Docs/Internal.hs:354)
    headers (src/Servant/Docs/Internal.hs:354)
    notes (src/Servant/Docs/Internal.hs:354)
    params (src/Servant/Docs/Internal.hs:354)
    rqtypes (src/Servant/Docs/Internal.hs:354)
    rqbody (src/Servant/Docs/Internal.hs:354)
    response (src/Servant/Docs/Internal.hs:354)
Warning: Servant.Docs.Internal: could not find link destinations for:
    Method <>~ .~ methodPost Hashable hashWithSalt hash HashMap MediaType Header Lens' Iso' Omega
Warning: Servant.Docs.Internal.Pretty: could not find link destinations for:
    MediaType ToJSON
Warning: Servant.Docs: could not find link destinations for:
    .~ Lens' HashMap Iso' MediaType Method <>~ methodPost Hashable hashWithSalt hash Header
Documentation created:
/tmp/dist-test.Jo0m/dist-newstyle/build/x86_64-linux/ghc-8.6.1/servant-docs-0.11.2/doc/html/servant-docs/index.html
Preprocessing library for servant-client-core-0.14.1..
Building library for servant-client-core-0.14.1..
Preprocessing library for servant-client-core-0.14.1..
Running Haddock on library for servant-client-core-0.14.1..
Warning: The documentation for the following packages are not installed. No
links will be generated to these packages: StateVar-1.1.1.1, aeson-1.4.1.0,
attoparsec-0.13.2.2, attoparsec-iso8601-1.0.1.0, base-compat-0.10.4,
base-orphans-0.8, base64-bytestring-1.0.0.1, bifunctors-5.5.3,
case-insensitive-1.2.0.11, comonad-5.0.4, contravariant-1.5, cookie-0.4.4,
data-default-class-0.1.2.0, distributive-0.6, dlist-0.8.0.5,
exceptions-0.10.0, free-5.1, generics-sop-0.3.2.0, hashable-1.2.7.0,
http-api-data-0.3.10, http-media-0.7.1.3, http-types-0.12.2,
integer-logarithms-1.0.2.2, mmorph-1.1.2, network-uri-2.6.1.0,
primitive-0.6.4.0, profunctors-5.3, random-1.1, safe-0.3.17,
scientific-0.3.6.2, semigroupoids-5.3.1, semigroups-0.18.5,
singleton-bool-0.1.4, string-conversions-0.4.0.1, tagged-0.8.6,
th-abstraction-0.2.8.0, time-locale-compat-0.1.1.5, transformers-base-0.4.5.2,
transformers-compat-0.6.2, unordered-containers-0.2.9.0, utf8-string-1.0.1.1,
uuid-types-1.0.3, vault-0.3.1.2, vector-0.12.0.1
Haddock coverage:
  71% (  5 /  7) in 'Servant.Client.Core.Internal.BaseUrl'
  Missing documentation for:
    Module header
    InvalidBaseUrlException (src/Servant/Client/Core/Internal/BaseUrl.hs:92)
Warning: 'client' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
  62% (  5 /  8) in 'Servant.Client.Core.Internal.Generic'
  Missing documentation for:
    Module header
    ClientList (src/Servant/Client/Core/Internal/Generic.hs:131)
    GClientList (src/Servant/Client/Core/Internal/Generic.hs:135)
  29% (  4 / 14) in 'Servant.Client.Core.Internal.Request'
  Missing documentation for:
    Module header
    RequestF (src/Servant/Client/Core/Internal/Request.hs:62)
    Request (src/Servant/Client/Core/Internal/Request.hs:72)
    GenResponse (src/Servant/Client/Core/Internal/Request.hs:84)
    Response (src/Servant/Client/Core/Internal/Request.hs:91)
    StreamingResponse (src/Servant/Client/Core/Internal/Request.hs:92)
    defaultRequest (src/Servant/Client/Core/Internal/Request.hs:95)
    appendToPath (src/Servant/Client/Core/Internal/Request.hs:106)
    appendToQueryString (src/Servant/Client/Core/Internal/Request.hs:110)
    addHeader (src/Servant/Client/Core/Internal/Request.hs:118)
   0% (  0 /  2) in 'Servant.Client.Core.Internal.ClientF'
  Missing documentation for:
    Module header
    ClientF (src/Servant/Client/Core/Internal/ClientF.hs:6)
 100% (  2 /  2) in 'Servant.Client.Core.Internal.BasicAuth'
 100% (  4 /  4) in 'Servant.Client.Core.Internal.Auth'
  25% (  1 /  4) in 'Servant.Client.Core.Internal.RunClient'
  Missing documentation for:
    RunClient (src/Servant/Client/Core/Internal/RunClient.hs:31)
    checkContentTypeHeader (src/Servant/Client/Core/Internal/RunClient.hs:37)
    decodedAs (src/Servant/Client/Core/Internal/RunClient.hs:45)
Warning: 'Capture' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'Header' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'QueryParam' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'Method' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'Response' is ambiguous. It is defined
    * at src/Servant/Client/Core/Internal/Request.hs:84:22
    * at src/Servant/Client/Core/Internal/Request.hs:91:1
    You may be able to disambiguate the identifier by qualifying it or
    by hiding some imports.
    Defaulting to 'Response' defined at src/Servant/Client/Core/Internal/Request.hs:84:1
Warning: 'ReqBody' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
Warning: 'ToJSON' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
  83% (  5 /  6) in 'Servant.Client.Core.Internal.HasClient'
  Missing documentation for:
    Module header
  70% ( 26 / 37) in 'Servant.Client.Core'
  Missing documentation for:
    Request (src/Servant/Client/Core/Internal/Request.hs:72)
    RequestF (src/Servant/Client/Core/Internal/Request.hs:62)
    defaultRequest (src/Servant/Client/Core/Internal/Request.hs:95)
    Response (src/Servant/Client/Core/Internal/Request.hs:91)
    GenResponse (src/Servant/Client/Core/Internal/Request.hs:84)
    RunClient (src/Servant/Client/Core/Internal/RunClient.hs:31)
    InvalidBaseUrlException (src/Servant/Client/Core/Internal/BaseUrl.hs:92)
    StreamingResponse (src/Servant/Client/Core/Internal/Request.hs:92)
    addHeader (src/Servant/Client/Core/Internal/Request.hs:118)
    appendToQueryString (src/Servant/Client/Core/Internal/Request.hs:110)
    appendToPath (src/Servant/Client/Core/Internal/Request.hs:106)
  79% ( 15 / 19) in 'Servant.Client.Core.Reexport'
  Missing documentation for:
    Response (src/Servant/Client/Core/Internal/Request.hs:91)
    StreamingResponse (src/Servant/Client/Core/Internal/Request.hs:92)
    GenResponse (src/Servant/Client/Core/Internal/Request.hs:84)
    InvalidBaseUrlException (src/Servant/Client/Core/Internal/BaseUrl.hs:92)
  25% (  1 /  4) in 'Servant.Client.Free'
  Missing documentation for:
    Module header
    client (src/Servant/Client/Free.hs:17)
    ClientF (src/Servant/Client/Core/Internal/ClientF.hs:6)
  75% (  3 /  4) in 'Servant.Client.Generic'
  Missing documentation for:
    Module header
Warning: Servant.Client.Core.Internal.BaseUrl: could not find link destinations for:
    FromJSON parseJSON Value Parser parseJSONList FromJSONKey fromJSONKey FromJSONKeyFunction fromJSONKeyList ToJSON toJSON toEncoding Encoding toJSONList toEncodingList ToJSONKey toJSONKey ToJSONKeyFunction toJSONKeyList MonadThrow
Warning: Servant.Client.Core.Internal.Generic: could not find link destinations for:
    Generic Code NP I
Warning: Servant.Client.Core.Internal.Request: could not find link destinations for:
    MediaType QueryItem Header Method Status HeaderName
Warning: Servant.Client.Core.Internal.RunClient: could not find link destinations for:
    Free MediaType
Warning: Servant.Client.Core: could not find link destinations for:
    QueryItem MediaType Header Method Generic Code Status Free FromJSON parseJSON Value Parser parseJSONList FromJSONKey fromJSONKey FromJSONKeyFunction fromJSONKeyList ToJSON toJSON toEncoding Encoding toJSONList toEncodingList ToJSONKey toJSONKey ToJSONKeyFunction toJSONKeyList MonadThrow HeaderName
Warning: Servant.Client.Core.Reexport: could not find link destinations for:
    Status Header Generic Code MediaType FromJSON parseJSON Value Parser parseJSONList FromJSONKey fromJSONKey FromJSONKeyFunction fromJSONKeyList ToJSON toJSON toEncoding Encoding toJSONList toEncodingList ToJSONKey toJSONKey ToJSONKeyFunction toJSONKeyList MonadThrow
Warning: Servant.Client.Free: could not find link destinations for:
    Free
Documentation created:
/tmp/dist-test.Jo0m/dist-newstyle/build/x86_64-linux/ghc-8.6.1/servant-client-core-0.14.1/doc/html/servant-client-core/index.html
Preprocessing library for servant-client-0.14..
Running Haddock on library for servant-client-0.14..
Warning: The documentation for the following packages are not installed. No
links will be generated to these packages: StateVar-1.1.1.1, aeson-1.4.1.0,
async-2.2.1, attoparsec-0.13.2.2, attoparsec-iso8601-1.0.1.0,
base-compat-0.10.4, base-orphans-0.8, base64-bytestring-1.0.0.1,
basement-0.0.8, bifunctors-5.5.3, blaze-builder-0.4.1.0,
case-insensitive-1.2.0.11, comonad-5.0.4, contravariant-1.5, cookie-0.4.4,
data-default-class-0.1.2.0, distributive-0.6, dlist-0.8.0.5,
exceptions-0.10.0, free-5.1, generics-sop-0.3.2.0, hashable-1.2.7.0,
http-api-data-0.3.10, http-client-0.5.13.1, http-media-0.7.1.3,
http-types-0.12.2, integer-logarithms-1.0.2.2, memory-0.14.18,
mime-types-0.1.0.8, mmorph-1.1.2, monad-control-1.0.2.3, network-2.7.0.2,
network-uri-2.6.1.0, primitive-0.6.4.0, profunctors-5.3, random-1.1,
safe-0.3.17, scientific-0.3.6.2, semigroupoids-5.3.1, semigroups-0.18.5,
singleton-bool-0.1.4, streaming-commons-0.2.1.0, string-conversions-0.4.0.1,
tagged-0.8.6, th-abstraction-0.2.8.0, time-locale-compat-0.1.1.5,
transformers-base-0.4.5.2, transformers-compat-0.6.2,
unordered-containers-0.2.9.0, utf8-string-1.0.1.1, uuid-types-1.0.3,
vault-0.3.1.2, vector-0.12.0.1, zlib-0.6.2
Haddock coverage:
Warning: 'Manager' is out of scope.
    If you qualify the identifier, haddock can try to link it
    it anyway.
  42% (  5 / 12) in 'Servant.Client.Internal.HttpClient'
  Missing documentation for:
    Module header
    runClientM (src/Servant/Client/Internal/HttpClient.hs:141)
    performRequest (src/Servant/Client/Internal/HttpClient.hs:144)
    performStreamingRequest (src/Servant/Client/Internal/HttpClient.hs:176)
    clientResponseToResponse (src/Servant/Client/Internal/HttpClient.hs:191)
    requestToClientRequest (src/Servant/Client/Internal/HttpClient.hs:199)
    catchConnectionError (src/Servant/Client/Internal/HttpClient.hs:241)
  88% (  7 /  8) in 'Servant.Client'
  Missing documentation for:
    runClientM (src/Servant/Client/Internal/HttpClient.hs:141)
Warning: Servant.Client.Internal.HttpClient: could not find link destinations for:
    Manager CookieJar MonadCatch catch MonadThrow throwM Alt <!> some many MonadBase liftBase MonadBaseControl liftBaseWith RunInBase restoreM StM Response Request
Warning: Servant.Client: could not find link destinations for:
    Manager MonadCatch catch MonadThrow throwM Alt <!> some many MonadBase liftBase MonadBaseControl liftBaseWith RunInBase restoreM StM CookieJar
Documentation created:
/tmp/dist-test.Jo0m/dist-newstyle/build/x86_64-linux/ghc-8.6.1/servant-client-0.14/doc/html/servant-client/index.html
The command "if $HADDOCK; then cabal new-haddock -w ${HC} ${TEST} ${BENCH} all; else echo "Skipping haddock generation";fi" exited with 0.
0.00s
0.00s
build-installed
Building without installed constraints for packages in global-db...
ed with 0.
0.00s$ if $UNCONSTRAINED; then rm -f cabal.project.local; echo cabal new-build -w ${HC} --disable-tests --disable-benchmarks all; else echo "Not building without installed constraints"; fi
cabal new-build -w ghc-8.6.1 --disable-tests --disable-benchmarks all
The command "if $UNCONSTRAINED; then rm -f cabal.project.local; echo cabal new-build -w ${HC} --disable-tests --disable-benchmarks all; else echo "Not building without installed constraints"; fi" exited with 0.
0.00s
before_cache.1
0.00s$ rm -fv $HOME/.cabal/packages/hackage.haskell.org/build-reports.log
before_cache.2
0.00s$ rm -fv $HOME/.cabal/packages/hackage.haskell.org/00-index.*
before_cache.3
0.00s$ rm -fv $HOME/.cabal/packages/hackage.haskell.org/*.json
removed ‘/home/travis/.cabal/packages/hackage.haskell.org/mirrors.json’
removed ‘/home/travis/.cabal/packages/hackage.haskell.org/root.json’
removed ‘/home/travis/.cabal/packages/hackage.haskell.org/snapshot.json’
removed ‘/home/travis/.cabal/packages/hackage.haskell.org/timestamp.json’
before_cache.4
0.00s$ rm -fv $HOME/.cabal/packages/hackage.haskell.org/01-index.cache
removed ‘/home/travis/.cabal/packages/hackage.haskell.org/01-index.cache’
before_cache.5
0.09s$ rm -fv $HOME/.cabal/packages/hackage.haskell.org/01-index.tar
removed ‘/home/travis/.cabal/packages/hackage.haskell.org/01-index.tar’
before_cache.6
0.00s$ rm -fv $HOME/.cabal/packages/hackage.haskell.org/01-index.tar.idx
removed ‘/home/travis/.cabal/packages/hackage.haskell.org/01-index.tar.idx’
before_cache.7
0.00s$ rm -rfv $HOME/.cabal/packages/head.hackage
cache.2
store build cache
0.00s
21.63schange detected (content changed, file is created, or file is deleted):
/home/travis/.cabal/packages/hackage.haskell.org/adjunctions/4.4/adjunctions-4.4.tar.gz
/home/travis/.cabal/packages/hackage.haskell.org/aeson/1.4.1.0/aeson-1.4.1.0.tar.gz
/home/travis/.cabal/packages/hackage.haskell.org/aeson-compat/0.3.9/aeson-compat-0.3.9.tar.gz
/home/travis/.cabal/packages/hackage.haskell.org/aeson-pretty/0.8.7/aeson-pretty-0.8.7.tar.gz
/home/travis/.cabal/packages/hackage.haskell.org/ansi-terminal/0.8.1/ansi-terminal-0.8.1.tar.gz
/home/travis/.cabal/packages/hackage.haskell.org/ansi-wl-pprint/0.6.8.2/ansi-wl-pprint-0.6.8.2.tar.gz
/home/travis/.cabal/packages/hackage.haskell.org/appar/0.1.4/appar-0.1.4.tar.gz
/home/travis/.cabal/packages/hackage.haskell.org/asn1-encoding/0.9.5/asn1-encoding-0.9.5.tar.gz
/home/travis/.cabal/packages/hackage.haskell.org/asn1-parse/0.9.4/asn1-parse-0.9.4.tar.gz
/home/travis/.cabal/packages/hackage.haskell.org/asn1-types/0.3.2/asn1-types-0.3.2.tar.gz
/home/travis/.cabal/packages/hackage.haskell.org/async/2.2.1/async-2.2.1.tar.gz
/home/t
...
changes detected, packing new archive
.
.
.
uploading archive
Done. Your build exited with 1.
```

fosskers commented:
Here's one tidbit:
```
Finished in 0.3275 seconds
33 examples, 0 failures
Test suite spec: PASS
Test suite logged to:
/tmp/dist-test.Jo0m/dist-newstyle/build/x86_64-linux/ghc-8.6.1/servant-client-0.14/t/spec/test/servant-client-0.14-spec.log
1 of 1 test suites (1 of 1 test cases) passed.
Configuring test suite 'readme' for servant-client-0.14..
Preprocessing test suite 'readme' for servant-client-0.14..
Building test suite 'readme' for servant-client-0.14..
[1 of 1] Compiling Main             ( README.lhs, /tmp/dist-test.Jo0m/dist-newstyle/build/x86_64-linux/ghc-8.6.1/servant-client-0.14/t/readme/build/readme/readme-tmp/Main.o )
Linking /tmp/dist-test.Jo0m/dist-newstyle/build/x86_64-linux/ghc-8.6.1/servant-client-0.14/t/readme/build/readme/readme ...
Running 1 test suites...
Test suite readme: RUNNING...
Error: ConnectionError "HttpExceptionRequest Request {\n  host                 = \"localhost\"\n  port                 = 8081\n  secure               = False\n  requestHeaders       = [(\"Accept\",\"application/json;charset=utf-8,application/json\")]\n  path                 = \"/books\"\n  queryString          = \"\"\n  method               = \"GET\"\n  proxy                = Nothing\n  rawBody              = False\n  redirectCount        = 10\n  responseTimeout      = ResponseTimeoutDefault\n  requestVersion       = HTTP/1.1\n}\n (ConnectionFailure Network.Socket.connect: <socket: 3>: does not exist (Connection refused))"
Test suite readme: PASS
Test suite logged to:
/tmp/dist-test.Jo0m/dist-newstyle/build/x86_64-linux/ghc-8.6.1/servant-client-0.14/t/readme/test/servant-client-0.14-readme.log
1 of 1 test suites (1 of 1 test cases) passed.
```
